internal/dataplane/sendconfig/kong.go

```diff
@@ -40,6 +57,8 @@ func New(
 	dbMode string,
 	concurrency int,
 	filterTags []string,
+	skipCACertificates bool,
+	enableReverseSync bool,
```

Comment:
If we've already extracted the config struct why not accept it here instead of a (pretty long) list of parameters?

Reply:
Yup, good point 👍